package.json

```
@@ -4,8 +4,8 @@
   "version": "1.1.1",
   "description": "💊 Valtio makes proxy-state simple for React and Vanilla",
   "main": "./index.js",
-  "module": "./esm/index.mjs",
-  "types": "index.d.ts",
+  "module": "./esm/index.js",
```

One more: "type" is missing. it's in https://github.com/pmndrs/zustand/pull/507/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R8

```suggestion
  "module": "./esm/index.js",
  "type": "commonjs",
```
We forgot to add in #184?

---

Not forgot, 
during the zustand fixes I was testing out every possible scenario for that issue and was constantly change the type of the outer package and inner so everything was forced, since I understood the layout by the time I worked with valtio I didn't add it in since it's defaulted to commonjs and the `esm/` folder has a forced `type:module` anyway.

---

cool. that's what I had expected. so, we can remove it in https://github.com/pmndrs/zustand/pull/507 ?

---

done